blocks_vertical/control.js

```diff
@@ -61,8 +61,7 @@ Blockly.Blocks['control_repeat'] = {
       "args0": [
         {
           "type": "input_value",
-          "name": "TIMES",
```

Comment:
But this still can't take a boolean, right?  So it has to have a check, but should be expanded to string || number